.eslintrc.js

```diff
@@ -0,0 +1,330 @@
+        "prefer-object-spread": "off",               // prefer use of an object spread over Object.assign
+        "prefer-promise-reject-errors": "error",     // require using Error objects as Promise rejection reasons
+        "prefer-regex-literals": "error",            // disallow the use of the RegExp constructor function with string literals as its arguments
+        "prefer-rest-params": "error",               // disallow usage of arguments variables
+        "prefer-spread": "error",                    // disallow usage of Function.prototype.apply() in situations where spread syntax could be used instead
+        "prefer-template": "error",                  // disallow usage of + operators with strings
+        "quote-props": [                             // require quotes around object literal property names
+            "error",
+            "consistent"                             // enforces a consistent quote style; in a given object, either all of the properties should be quoted, or none of the properties should be quoted
+        ],
+        "quotes": "off",                             // enforces the consistent use of either backticks, double, or single quotes
+        "radix": [                                   // requires passing the radix parameter when using parseInt()
+            "error",
+            "as-needed"                              // disallows providing the 10 radix
+        ],
+        "require-atomic-updates": "error",           // disallow assignments that can lead to race conditions due to usage of await or yield
+        "require-await": "error",                    // warns async functions which have no await expression
+        "require-unicode-regexp": "off",             // enforce the use of `u` flag on regular expressions
+        "rest-spread-spacing": "error",              // enforce consistent spacing between rest and spread operators and their expressions
+        "semi": "error",                             // requires semicolons at the end of statements
+        "semi-spacing": "error",                     // enforce spacing around a semicolon. Default: {"before": false, "after": true}
+        "semi-style": [                              // enforces that semicolons are at the end of statements
+            "error",
+            "last"
+        ],
+        "sort-imports": "error",                     // requires all import declarations and verifies that all imports are first sorted by the used member syntax and then alphabetically by the first member
+        "sort-keys": "off",                          // requires all property definitions of object expressions and verifies that all variables are sorted alphabetically
+        "sort-vars": "error",                        // requires all variable declaration blocks and verifies that all variables are sorted alphabetically
```

Comment:
unclear, should it be
```suggestion
        "sort-vars": "error",                        // requires that all variable names in declaration blocks be sorted alphabetically
```
the other `sort-` descriptions are similarly unclear

Reply:
Fixed the description of both rules